src/proto-gen/proto-gen.cmake

```diff
@@ -17,24 +17,24 @@
 function(add_proto_source name)
     add_custom_command(
         OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/proto-gen/${name}.pb.cc
-        DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/proto/${name}.proto
+        DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/milvus-proto/${name}.proto
```

Comment:
There is still a subfolder `proto` under `milvus-proto`.

Reply:
right..... should be ok now